controllers/managedocs_controller.go

```diff
@@ -1048,6 +1059,16 @@ func (r *ManagedOCSReconciler) reconcilePrometheus() error {
 			},
 			Key: alertRelabelConfigSecretKey,
 		}
+		// Enabling remote write for only provider clusters
+		if r.DeploymentType == providerDeploymentType {
+			var remoteWriteSpec *promv1.RemoteWriteSpec
+			if remoteWriteSpec, err = r.reconcilePrometheusRemoteWrite(); remoteWriteSpec != nil {
```

Comment:
- why not below, as var will be available through out the conditional block and I don't see it being used after that?
- iirc, even when you are appending to slice it only takes the copy so separately declaring var isn't required
```suggestion
			if remoteWriteSpec, err := r.reconcilePrometheusRemoteWrite(); remoteWriteSpec != nil {
```
- having said that, it's not mandatory to include this comment if you aren't convinced

Reply:
Leftover from previous implementation. Thanks for pointing it out!

Reply:
Done